src/my_react/render.ts

```diff
@@ -1,20 +1,37 @@
+import { camelCaseToKebabCase } from 'src/utils/caseConverter';
+
+const handleAttribute = ($elem: HTMLElement, props: JSX.Props, prop: string) => {
+  /**
+   * prop이 boolean으로 들어가야 하는 상황 생길 때마다 케이스 추가 필요
+   */
+  if (prop === 'checked') {
+    const $input = $elem as HTMLInputElement;
+    $input.checked = props[prop];
+  }
+
+  $elem.setAttribute(prop === 'className' ? 'class' : camelCaseToKebabCase(prop), props[prop]);
+};
+
+const addEventHandler = ($elem: HTMLElement, eventType: string, handler: (e: Event) => void) => {
+  $elem.removeEventListener(eventType, handler); // 이벤트 핸들러 중복 등록 방지를 위해 먼저 지우고 달기
+  $elem.addEventListener(eventType, handler);
+};
+
 const addAttribute = ($elem: HTMLElement, props: JSX.Props, prop: string) => {
   /* props의 종류에 따라 필요한 처리, 예외 케이스 더 확인되면 추가 필요 */
   switch (prop) {
+    case 'key':
+    case 'children':
     case 'styles': {
-      Object.keys(props[prop]).forEach((key) => {
-        ($elem.style as any)[key] = props[prop][key];
+      Object.entries(props[prop]).forEach((entries) => {
+        ($elem.style as any)[camelCaseToKebabCase(entries[0])] = entries[1];
```

Comment:
[P4] forEach에서 구조분해를 하면 더 깔끔할 것 같아요!
```suggestion
      Object.entries(props[prop]).forEach(([key, value]) => {
        ($elem.style as any)[camelCaseToKebabCase(key)] = value;
```